yafti/app.py

```diff
@@ -33,6 +37,15 @@ def __init__(self, cfg: Config = None, loop=None):
     def run(self, *args, **kwargs):
         configured_mode = self.config.properties.mode
         _p = self.config.properties.path.expanduser()
+        ## ---- TODO: START: REMOVE THIS HACK AFTER ENOUGH PEOPLE HAVE UPGRADED ----
```

Comment:
```suggestion
        # TODO(GH-#): Remove this prior to 1.0 release
```

For todos that support some form of compatibility, I'd prefer we instead open a github issue describing what to remove and use that as `GH-#` where `#` is the issue id. That helps to provide more indepth context for those looking at the code about why this was added and when it should be removed. I think the 1.0 target is the most ideal for removal as it's our chance to break backwards compat and move forward. My guess is we're a few months away from 1.0 after the yafti.core rewrite and separation of runner logic from display logic.

Reply:
Yeah that's a good idea, I'm opening a ticket to track it and pointing the code comment at it now.